samples/unit-tests/indicator-price-indicator/recalculations/demo.js

```diff
@@ -318,4 +318,68 @@ QUnit.test('The lastPrice color, #15074.', function (assert) {
         '#00ff00',
         'The lastPrice color should remain the same after toggle.'
     );
+});
+
+QUnit.test('The currentPriceIndicator for multiple series, #14888.', function (assert) {
+    const chart = Highcharts.stockChart('container', {
+            yAxis: [{
+                height: '60%'
+            }, {
+                top: '65%',
+                height: '35%'
+            }],
+            stockTools: {
+                gui: {
+                    enabled: true,
+                    buttons: [
+                        'currentPriceIndicator'
+                    ]
+                }
+            },
+            series: [{
+                id: 'main',
+                color: '#00ffff',
+                data: [2, 6, 8, 6, 3, 1, 1, 3, 5, 6, 9, 9, 9, 7, 4, 2, 1, 9]
+            }, {
+                type: 'sma',
+                linkedTo: 'main',
+                color: '#ff00ff'
+            }, {
+                type: 'column',
+                color: '#000000',
+                yAxis: 1,
+                data: [10, 2, 5, 6, 1, 3, 5, 1, 3, 5, 4, 1, 3, 5, 6, 4, 1, 4]
+            }]
+        }),
+        button = chart.stockTools.listWrapper.childNodes[0].childNodes[0];
+
+    // Click the currentPriceIndicator button in the stock tools.
+    chart.navigationBindings.options.bindings.currentPriceIndicator.init.call(
+        chart.navigationBindings, button);
+
+    chart.series.forEach(function (serie) {
+        if (serie.options.id !== 'highcharts-navigator-series') {
+            assert.strictEqual(
+                serie.lastPrice.attr('stroke'),
+                serie.color,
+                'Each series lastPrice line should have color as series.'
+            );
+            assert.strictEqual(
+                serie.crossLabel.attr('fill'),
+                serie.color,
```

Comment:
The single form of "series" is "series. I've got my own crusade to correct it in the code wherever I can.

I am aware there was a discussion about it a while ago, but I still think that we should be consistent with a correct form and 95%+ of "series" cases, not "serie".

Reply:
You are right, we should be consistent with the names, so I will join your crusade and I will try to change it everywhere.